src/main/java/com/needle/FsoFso/admin/dao/SellerProductDaoImpl.java

```diff
@@ -0,0 +1,29 @@
+package com.needle.FsoFso.admin.dao;
+
+import java.util.List;
+
+import org.apache.ibatis.session.SqlSession;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+
+import com.needle.FsoFso.admin.dto.SellerProductDto;
+
+@Repository
+public class SellerProductDaoImpl implements SellerProductDao{
+
+	@Autowired
+	SqlSession session;
+	
+	String ns = "SellerProduct.";
```

Comment:
namespace 는 상수로 사용해도 되지 않을까요?

```suggestion
	private static final String NAME_SPACE = "SellerProduct.";
```